idsvr/templates/hpa.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.autoscaling.enabled }}
+apiVersion: autoscaling/v2beta1
```

Comment:
I think you need to use v2beta2 in order for custom metrics to work. See [this post](https://stackoverflow.com/questions/54459447/difference-between-api-versions-v2beta1-and-v2beta2-in-horizontal-pod-autoscaler).

Reply:
you are correct, nice catch :)